lib/tasks/db.rake

```diff
@@ -265,6 +265,12 @@ namespace :db do
     DBMigrator.from_config(RakeConfig.config, db_logger).rollback(number_to_rollback)
   end
 
+  def logging_output
```

Comment:
The task `ensure_migrations_are_current` also calls `Steno.init`. This can be replaced with `logging_output` as well.